forms-flow-bpm/Dockerfile

```diff
@@ -19,16 +18,14 @@ COPY forms-flow-bpm-utils/src ./forms-flow-bpm-utils/src/
 RUN mvn -s /usr/share/maven/ref/settings-docker.xml install -P camunda
 
 # Final custom slim java image (for apk command see 17-jdk-alpine-slim)
-FROM openjdk:17-jdk-alpine
-# Update packages including OpenSSL
-RUN apk update && apk upgrade
+FROM openjdk:21-ea-jdk
 
 # set label for image
 LABEL Name="formsflow"
 
-ENV JAVA_VERSION=17-ea+14
+ENV JAVA_VERSION=21-ea+14
 ENV JAVA_HOME=/opt/java/openjdk-17\
```

Comment:
@sumesh-aot Do we need to change from 17 to 21 here ?

Reply:
good catch, updated